cli/src/tree.rs

```diff
@@ -30,23 +30,23 @@ fn populate_node(
         }
 
         for ingredient in manifest.ingredients().iter() {
+            let title = ingredient.title().unwrap_or("Untitled");
```

Comment:
Is `Untitled` a default that will be reused everywhere?

Reply:
good question. I don't think we have a UI call on that, but this is only for the tree view in c2patool.